docs/using-data.md

```diff
@@ -2,6 +2,40 @@
 
 All of the ClearlyDefined data is available for everyone to see and use. You can browse and inspect in a [convenient web ui](#website) or hook up a client to the [REST API](https://api.clearlydefined.io/api-docs/) and integrate it into your systems.
 
+## REST API Usage Considerations
+
+ClearlyDefined enforces specific rate limits on REST API calls to ensure stability and integrity of the platform. 
+
+You can check the _x-ratelimit-limit_ and _x-ratelimit-remaining_ response headers to track your usage and reset window.
+
+If the rate limit is reached, an _HTTP 429_ response code will be returned. The rate limit is based on the window below.
+
+### Production Instance Rate Limits
+
+Host: [api.clearlydefined.io](https://api.clearlydefined.io/)
+\([Docs](https://api.clearlydefined.io/api-docs/)\)
+
+| Endpoint      | Method  | Limit    | Window |
+|---------------|---------|----------|--------|
+| /definitions  | POST    | 250 /min | 1 min  |
+| /curations    | POST    | 250 /min | 1 min  |
+| /notices      | POST    | 250 /min | 1 min  |
+
+All other endpoints are max 2K requests per minute. 
+
+### Development Instance Rate Limits
+
+Host: [dev-api.clearlydefined.io](https://dev-api.clearlydefined.io/)
+\([Docs](https://dev-api.clearlydefined.io/api-docs/)\)
+
+| Endpoint      | Method  | Limit    | Window |
+|---------------|---------|----------|--------|
+| /definitions  | POST    | 250 /min | 5 min  |
+| /curations    | POST    | 250 /min | 5 min  |
+| /notices      | POST    | 250 /min | 5 min  |
+
```

Comment:
As per Manny's comment: 
> For DEV: The routes above are correct (250req/ 1min window). For all other routes in dev, the rate is max of 500 requests within a 5 minute window
> 
So the window is 1 min

Reply:
Alright, I updated the table format to be closer to what I originally had. Should be good now.